test/mrb_exec.rb

```diff
@@ -28,3 +28,14 @@
   assert_true(p.is_a?(Integer))
   assert_true(status.success?)
 end
+
+assert("Kernel#exec sys fail") do
+  expected_error_message = "sys failed. errno: 2 message: No such file or directory mrbgem message: execv failed"
+
+  begin
+    # always failed
+    p = Process.fork { exec "/bin/bbaash" }
```

Comment:
Thank you for your review. I fixed via https://github.com/haconiwa/mruby-exec/pull/8/commits/b3cae2cb7e3c8ff07b0b428ab2fc471dc7dd3553